<?xml version="1.0"?>
<launch>
  <node pkg="BP" type="detector" name="detector" output="screen">
    <remap from="kinect_data" to="/camera/rgb/image_raw" />
    <remap from="detect_balls" to="/ball_detections" />
  </node>
</launch>
